src/lib.rs

```diff
@@ -276,20 +297,39 @@ impl Element {
 
     /// Find a child element with the given name and return a reference to it.
     pub fn get_child<P: ElementPredicate>(&self, k: P) -> Option<&Element> {
-        self.children.iter().find(|e| k.match_element(e))
+        self.children
+            .iter()
+            .filter_map(|e| match e {
+                XMLNode::Element(elem) => Some(elem),
+                _ => None,
+            })
+            .find(|e| k.match_element(e))
```

Comment:
should probably provide a way to get at text, commen t, etc nodes

Reply:
Agreed 👍